util/helm/client_test.go

```diff
@@ -312,14 +310,12 @@ func TestGetTagsFromURLEnvironmentAuthentication(t *testing.T) {
 
 		assert.Equal(t, expectedAuthorization, authorization)
 
-		responseTags := TagsList{
-			Tags: []string{
-				"2.8.0",
-				"2.8.0-prerelease",
-				"2.8.0_build",
-				"2.8.0-prerelease_build",
-				"2.8.0-prerelease.1_build.1234",
-			},
+		responseTags := []string{
+			"2.8.0",
+			"2.8.0-prerelease",
+			"2.8.0_build",
+			"2.8.0-prerelease_build",
+			"2.8.0-prerelease.1_build.1234",
 		}
```

Comment:
For testing purposes we can add a private `tagsList` type in the test itself and then revert to almost the same thing as before

```suggestion
		responseTags := tagsList{
			Tags: []string{
				"2.8.0",
				"2.8.0-prerelease",
				"2.8.0_build",
				"2.8.0-prerelease_build",
				"2.8.0-prerelease.1_build.1234",
			},
		}
```